### PR TITLE
Add a script to import books to the sdr api

### LINKED ISSUE #56: Deposit harvested books into SDR via deposit API
And track that the deposit was successful, recording this status in the existing relevant database column. Note that the deposit operation is asynchronous.

The first step of this is to get the catkey from dor-services-app using a barcode. See https://github.com/sul-dlss/dor-services-client/issues/124

Fields needed for deposit (for the most basic of cases)
1. Admin Policy (APO)
1. Collection 
1. Catkey 
1. Source ID (construct as `googlebooks:stanford_{barcode}`)
1. tags =  ["Project : GoogleBooks"]
1. other_ids = { barcode: 12345 } 

This work includes using the sdr-client to access the SDR API, and may involved modifying sdr-client to disentangle CLI concerns from library concerns (e.g., `puts`ing all over the place).

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

So that google books can be ingested into SDR.

Fixes #56

## Was the documentation (README, DevOpsDocs, wiki, openapi.json) updated?
yes.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,15 +7,22 @@ gem 'rails', '~> 6.0'
 
 gem 'config'
 # 0.16.2 provides on_data, which was removed in 0.17 but (should be ) re-added in 1.0. See https://github.com/lostisland/faraday/issues/1025
-# However, signet depends on faraday (~> 0.9) so can't use 1.0 yet.
+# However, signet depends on faraday (~> 0.9) and googleauth depends on faraday (~> 0.12)
+# so can't use 1.0 yet.
 gem 'faraday', '~> 0.16.2'
+
+# This is a branch of signet that supports faraday 1.0
+# gem 'signet', github: 'splitwise/signet', branch: 'jas14/update-faraday'
```
Comment (justinlittman) [inline Gemfile]: Cruft?

File: spec/services/sdr_importer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+RSpec.describe SdrImporter do
+  subject(:import) do
+    described_class.import(directory: 'app/assets/config', barcode: '20500021556')
+  end
+
+  let(:lookup_client) { instance_double(Dor::Services::Client::Marcxml, catkey: '10042225') }
+
+  before do
+    # allow(Settings.sdr_api).to receive(:email).and_return('jcoyne85@stanford.edu')
```
Comment (justinlittman) [inline spec/services/sdr_importer_spec.rb]: Cruft?

File: Gemfile (hunk 1 above)
Comment (jcoyne) [reply Gemfile]: Not so much, this is a hint to my future self on how we can use faraday 1.0

File: spec/services/sdr_importer_spec.rb (hunk 2 above)
Comment (jcoyne) [reply spec/services/sdr_importer_spec.rb]: yes.

**Pull request merged**